pages/models.py

```diff
@@ -988,6 +988,19 @@ class InitiativeEndorsementPage(Page):
         ImageChooserPanel('social_image')
     ]
 
+    def _get_initiative_title(self):
+        """Build title for initiative and support legacy pages"""
+        if self.initiative_title and self.initiative_name:
```

Comment:
What if one of these is blank/None?

Reply:
It'll just default to title which is suitable.